### PR TITLE
Med 8 add up down scroll

### PR DESCRIPTION
**Pull request by pavlo-serhiichuk**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import './index.css';
 import React, {lazy, Suspense} from "react";
 import {Route} from "react-router-dom";
-import {Wrapper, Content} from "./App.styles";
+import {Wrapper, Content, A} from "./App.styles";
```
Comment (SVladikO) [inline src/App.js]: A?

File: src/App.js [hunk 2]
```diff
@@ -23,7 +24,10 @@ function App({query, isSignInModalOpen, isLoginModalOpen, isMoreInfoModalOpen, i
     return (
         <>
             <Wrapper>
+                <a name={'top'}></a>
+                <A href={'#down'}><BsFillCaretDownFill className={'goToDown'}/></A>
```
Comment (SVladikO) [inline src/App.js]: Please use size attribute

File: src/App.js [hunk 3]
```diff
@@ -40,6 +44,8 @@ function App({query, isSignInModalOpen, isLoginModalOpen, isMoreInfoModalOpen, i
                         <Route path="/article" render={() => <FullArticleContainer id={query.get("id")}/>}/>
                     </Routers>
                 </Content>
+                <A name={'down'}></A>
+                <A href={"#top"}><BsFillCaretUpFill className='goToTop'/></A>
```
Comment (SVladikO) [inline src/App.js]: The same

File: src/index.css [hunk 4]
```diff
@@ -36,3 +36,23 @@ a {
 a:hover {
   color: #000000;
 }
+.goToDown, .goToTop {
```
Comment (SVladikO) [inline src/index.css]: Use this styles for as parent>*

**Review by SVladikO — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (pavlo-serhiichuk) [reply src/App.js]: A is a styled-component element

File: src/App.js (hunk 2 above)
Comment (pavlo-serhiichuk) [reply src/App.js]: fixed

File: src/App.js (hunk 3 above)
Comment (pavlo-serhiichuk) [reply src/App.js]: fixed

File: src/index.css (hunk 4 above)
Comment (pavlo-serhiichuk) [reply src/index.css]: fixed

File: src/App.styles.js [hunk 5]
```diff
@@ -14,10 +15,8 @@ export const Wrapper = styled.div`
 `
 
 export const Content = styled.div`
-      display: flex;
-      flex-direction: row;  
+    display: flex;
+    flex-direction: row;
```
Comment (SVladikO) [inline src/App.styles.js]: is it default value ?

File: src/common/QuickScrolls/QuickScrolls.component.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+            <a name={'top'}></a>
+            <SGoDown href={'#down'}><BsFillCaretDownFill style={iconStyle}/></SGoDown>
+        </>
+    )
+}
+
+export const GoTop = () => {
+    return (
+        <>
+            <a name={'down'}></a>
```
Comment (SVladikO) [inline src/common/QuickScrolls/QuickScrolls.component.js]: separate this row with SGoTop

File: src/common/QuickScrolls/QuickScrolls.component.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+            <SGoTop href={"#top"}><BsFillCaretUpFill style={iconStyle}/></SGoTop>
```
Comment (SVladikO) [inline src/common/QuickScrolls/QuickScrolls.component.js]: SGoTop bad name

**Review by SVladikO — APPROVED** [review]
Good job

**Pull request merged**